main.rb

```diff
@@ -26,6 +28,7 @@ class Main
       value:
         "**`~year <1-4, masters, alumni>`** - add your current academic status to your profile.\n"\
         "**`~purge <2-99>`** - remove the last `n` messages in channel (**admin only**)\n"\
+        "**`~latex <latex command>`** - write latex in Mathmode to get a rendered image.\n"\
```

Comment:
These are examples of things that are appearing as new changes but already exist in the current master branch. You've got some messed up history going on here.